runtime/compiler/control/J9Options.cpp

```diff
@@ -1328,26 +1328,6 @@ J9::Options::fePreProcess(void * base)
          }
       }
 
-
-   // The -Xlp option may have a numeric argument but we don't care what
-   // it is because it applies to large data pages.
-   //
-   if (FIND_ARG_IN_VMARGS(EXACT_MATCH, "-Xlp", 0) >= 0)
-      {
-      self()->setOption(TR_EnableLargePages);
-      self()->setOption(TR_EnableLargeCodePages);
-      }
-
-   int32_t lpArgIndex;
-   UDATA lpSize = 0;
-   char *lpOption = "-Xlp";
-   if ((lpArgIndex=FIND_ARG_IN_VMARGS(EXACT_MEMORY_MATCH, lpOption, 0)) >= 0)
-      {
-      GET_MEMORY_VALUE(lpArgIndex, lpOption, lpSize);
-      self()->setOption(TR_EnableLargePages);
-      self()->setOption(TR_EnableLargeCodePages);
-      }
```

Comment:
@gita-omr We can also remove setting the two options here. The size argument is only used once, so instead of extracting the size argument here I've moved it closer to where it is used.